### ISSUE TITLE
Very Low Validation Accuracy for Resnet50 and Resnet101 models using TF-TRT

### ISSUE THREAD

**Issue by Selventhiranraj** [tensorflow/tensorrt#258]
I'm getting 0.1% as Top 1 accuracy for the validation of Resnet50 and Resnet101 models using the TF-TRT.
Validation script: https://github.com/tensorflow/tensorrt/blob/master/tftrt/examples/image-classification/image_classification.py
Dataset : Imagenet
Model : https://tfhub.dev/tensorflow/resnet_50/classification/1
Preprocess method : vgg

Why is the accuracy very low?? Any help/suggestions appreciated. Thanks

![tftrt_results](https://user-images.githubusercontent.com/60138679/120789629-791d8c80-c54f-11eb-98b1-b5f39ccb6a36.PNG)

**Comment by DEKHTIARJonathan** [comment]
I'm unfortunately unable to reproduce your issue. We are routinely running these benchmarks and we never had such issue.

Here are a few ideas to investigate:

- try not to convert the model with TF-TRT and just use normal to TF to conduct inference:
```bash
./scripts/resnet_v1_50.sh \
    --use_xla \
    --data_dir=/data/imagenet/train-val-tfrecord --model_dir=/models
```
A) If that works perfectly, there might be an issue in TF-TRT.
B) If that doesn't, the issue comes from either: your checkpoint/SavedModel is bad or your dataloading.

- if A) above: try using our TF-TRT automated scripts:
```bash
./scripts/resnet_v1_50.sh \
    --use_xla \
    --data_dir=/data/imagenet/train-val-tfrecord --model_dir=/models \
    --use_tftrt --tftrt_precision="FP16"
```

If that doesn't solve the issue, I will need a complete reproducer case with step-by-step instructions in a docker container to reproduce the issue (you will need to provide the checkpoint).

**Comment by mingxiaoh** [comment]
meet same issue
